mods/parsoid.js

```diff
@@ -784,7 +784,8 @@ PSP.makeTransform = function(from, to) {
 
     return function(restbase, req) {
         var rp = req.params;
-        if (!req.body || !req.body[from]) {
+        if ((!req.body && req.body !== '')
```

Comment:
@gwicke ah, actually, no.

But we could add that by adding a `parameters` section to the internal endpoints in the spec.

But anyway, agreed, let's not block. I'll try to play with spec-based thing tomorrow